x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml

```diff
@@ -221,7 +221,7 @@ setup:
   - gt: {esql.functions.to_long: $functions_to_long}
   - match: {esql.functions.coalesce: $functions_coalesce}
   - gt: {esql.functions.categorize: $functions_categorize}
-  - length: {esql.functions: 134} # check the "sister" test above for a likely update to the same esql.functions length check
+  - length: {esql.functions: 135} # check the "sister" test above for a likely update to the same esql.functions length check
```

Comment:
i don't think we need to update this - I am adding the release tests label to check

Reply:
Hopefully those errors get corrected when registering the function under snapshot functions 🤞

Reply:
I think you'll need to add `match_phrase` capability to these tests, otherwise they will fail for release as the function will not be registered